librad/src/git/storage.rs

```diff
@@ -992,6 +992,12 @@ where
 
         Ok(())
     }
+
+    pub fn is_tracked(&self, urn: RadUrn, peer: PeerId) -> bool {
```

Comment:
Can be done in `O(1)` by `self.backend.find_remote(&tracking_remote_name(urn, peer))`. Also, passing references is preferable I think.

Reply:
Thanks. I had checked `find_remote` but had not seen `tracking_remote_name` :+1: